finagle-netty4/src/main/scala/com/twitter/finagle/netty4/channel/ServerBridge.scala

```diff
@@ -11,7 +11,9 @@ import io.netty.channel.ChannelHandler.Sharable
 private[netty4] class ServerBridge[In, Out](
     transportFac: Channel => Transport[In, Out],
     serveTransport: Transport[In, Out] => Unit)
-  extends ChannelInitializer[Channel] {
+  extends ChannelInboundHandlerAdapter {
+
+  override def handlerAdded(ctx: ChannelHandlerContext): Unit = {}
```

Comment:
I'm still not sure why we need this. Also, how does `channelInit` work given that it's now `ChannelInboundHandlerAdapter` and not `ChannelInitializer`?

How do you feel about creating a channel transport on `channelActive` (in this server bridge) and delaying the `channelActive` event instead, in your new `SslServerConnectHandler`?